tests/testthat/test-include-hba1c.R

```diff
@@ -0,0 +1,77 @@
+lab_forsker <- tibble::tribble(
+  ~patient_cpr, ~samplingdate, ~analysiscode, ~value,
+  "498718589800", "20230101", "NPU27300", 49,
+  "498718589801", "20230101", "NPU03835", 6.6,
+  "498718589802", "20230101", "NPU03835", 6.3,
+  "498718589803", "20230101", "NPU27300", 47,
+  # Duplicate patient_cpr but with the old units.
+  "498718589803", "20210101", "NPU27300", 49,
+  "498718589803", "20220101", "NPU03835", 6.5,
+  # Duplicate patient_cpr when old and new units are the same date.
+  "498718589805", "20000101", "NPU03835", 6.5,
+  "498718589805", "20000101", "NPU27300", 49,
+  # Duplicate but with old below threshold and new above it.
+  "498718589806", "20000101", "NPU03835", 6.3,
+  "498718589806", "20000101", "NPU27300", 49,
+  # Duplicate but with new below threshold and old above it.
+  "498718589807", "20200101", "NPU03835", 6.6,
+  "498718589807", "20200101", "NPU27300", 47,
```

Comment:
@Aastedet So here, if there is a duplicate date, with new below but old above the threshold, should that be counted as a true condition?

Reply:
Yes, that should be counted as a positive inclusion event.

Reply:
Whenever you approve this PR, I'll merge it in ☺️

Reply:
😉 😉 @signekb  😄 😄